checkfiles.sh

```diff
@@ -0,0 +1,18 @@
+#This script is taked from rAthena function.sh as a checker for files in forlder.
```

Comment:
Just fix a small typo
```suggestion
#This script is taked from rAthena function.sh as a checker for files in folder.
```